spec/features/player_tests_spec.rb

```diff
@@ -0,0 +1,24 @@
+feature 'Enter names' do
+    scenario 'submitting names' do
+      sign_in_and_play
+      expect(page).to have_content "Khalid"
+    end
+  end    
+
+  feature 'play game ' do
+    scenario 'click Rock' do
+      sign_in_and_play
+      click_button 'rock'
+      
+      expect(page).to have_content "confirm to see if you have won"
+    end
+
+    scenario 'click Rock' do
+      sign_in_and_play
+      click_button 'rock'
+      click_link 'ok'
+      
+      expect(page).to have_content "Khalid won"
+    end
```

Comment:
I think it would help to describe the difference between the two scenarios, they both say 'click rock'